src/renderer/shared/ui/Alert/common/constants.ts

```diff
@@ -5,16 +5,19 @@ export const IconName: Record<Variant, IconNames> = {
   info: 'info',
   warn: 'warn',
   error: 'warn',
+  success: 'checkmarkOutline',
 };
 
 export const ViewStyle: Record<Variant, string> = {
   info: 'bg-alert-background border-alert-border',
   warn: 'bg-alert-background-warning border-alert-border-warning',
   error: 'bg-alert-background-negative border-alert-border-negative',
+  success: 'bg-[rgba(1,_166,_62,_0.08)] border-icon-positive',
```

Comment:
There are no `background-positive/success`?

Reply:
There weren't cause we didn't had success state for alert. But design team already added iti.